### DISCUSSION TITLE
Add "refreshCells" support for ui.aggrid

### DISCUSSION THREAD

**Discussion by plundeen** [zauberzeug/nicegui#861] [Ideas / Feature Requests]
I'm working on a prototype UI that leverages the `ui.aggrid` to display tabular data, along with a `ui.timer` to periodically refresh the grid. Unfortunately, using the `.update()` method causes a few undesirable behaviors:

- there is noticeable UI flicker as the table is redrawn
- the scroll position is reset
- any custom sorting is reset

I suspect that could be address by adding support for `refreshCells` and `refreshRows` methods.
https://www.ag-grid.com/javascript-data-grid/view-refresh/

I imagine a lot of users would benefit from such a feature.

**Comment by plundeen** [comment]
Update: after posting, I realized I could probably call `run_method("refreshCells")` to achieve this effect. However, it doesn't appear to be refreshing as I had hoped. 

I'll go ahead and close this feature request, though, since I think the initial description is not valid. Maybe I'll open another to request more generically a way to update `aggrid` elements without the above issues.

**Reply by ftippett** [reply]
Did you find a solution for updating aggrid? I would rather not use Quasar tables.

**Comment by pythonormen** [comment]
I changed to ui.table / quasar for that reason. Do you really need aggrid?
